test/compliance/resources/srv/index.cds

```diff
@@ -0,0 +1,17 @@
+using {complex as my} from '../db/complex';
```

Comment:
You don't need a service. Please just add these projections into the data model.